### DISCUSSION TITLE
How to put a external link to home page

### DISCUSSION THREAD

**Discussion by cocobase** [facebook/docusaurus#8048] [General]
![截屏2022-09-04 下午10 46 27](https://user-images.githubusercontent.com/60334709/188319634-2693cad3-24d4-46ea-a568-d93aafeaa5f1.png)

I want to put a external link to home page below [feature list item]
and I modified source code 'index.tsx'

```
<div className={clsx('col col--4')}>
      <div className="text--center">
        <Svg className={styles.featureSvg} role="img" />
      </div>
      <div className="text--center padding-horiz--md">
        <h3>{title}</h3>
        <p>{description}</p>
      </div>
      <div>
        <a href="www.google.com" target="_blank">Google</a>
      </div>
    </div>
```

but it dosen't woks and show "page can't find"

**Comment by pranabdas** [comment]
Looks like if the url does not start with `https://` or `http://`, it is treated as local path. You can use `href="https://www.google.com"`. Probably, you don't want the same link for all features, in that case you can include the url in the `FeatureList` array description.

**Reply by cocobase** [reply]
Thanks for your advice, as your suggestion, it works very well. I put a link url start with https//  in the FeatureList array description, Now this link do not display for all features. This is what I needed.
